### PR TITLE
fix to support list of properties in propertysets[CMP-2659]

### PR DESCRIPTION
**Pull request by reach2shaik468**
Property Toolkit to support OneFuse_CreateProperties_<custom-id> having an array/list of values.

### CODE REVIEW

File: onefuse/admin.py [hunk 1]
```diff
@@ -765,13 +771,19 @@ def get_create_properties(self, template_properties: dict):
...
+
+                if isinstance(value_obj, str):
                     value_obj = json.loads(value_obj)
-                if value_obj["key"] and value_obj["value"]:
+
+                if isinstance(value_obj, list):
+                    for item in value_obj:
+                        if isinstance(item, dict) and "key" in item and "value" in item:
+                            create_properties[item["key"]] = item["value"]
+                elif isinstance(value_obj, dict) and "key" in value_obj and "value" in value_obj:
```
Comment (laltomar) [inline onefuse/admin.py:792]: This code seems very hard to follow, a few comments describing what you are doing will go a long way for other developers.

**Review by laltomar — CHANGES_REQUESTED** [review]
LGTM, took me a few minutes to actually figure out what you were doing, please add some comments for future developers.

File: onefuse/admin.py (hunk 1 above)
Comment (reach2shaik468) [reply onefuse/admin.py:792]: I added the comments based on your suggestions. Thank you.

**Review by laltomar — APPROVED** [review]
LGTM!
